app/helpers/content_helper.rb

```diff
@@ -53,45 +55,69 @@ def side_navigation(page)
     end
   end
 
+  # @return [Resource, OpenStruct]
   def signup
     Resource.by_name('ctas.signup') || null_resource('ctas.signup')
   end
 
+  # @return [Resource, OpenStruct]
   def not_found
     Resource.by_name('error.not_found') || null_resource('error.not_found')
   end
 
+  # @return [Resource, OpenStruct]
   def internal_server_error
     Resource.by_name('error.internal_server_error') || null_resource('error.internal_server_error')
   end
 
+  # @return [Resource, OpenStruct]
   def unprocessable_entity
     Resource.by_name('error.unprocessable_entity') || null_resource('error.unprocessable_entity')
   end
 
+  # @return [Resource, OpenStruct]
   def service_unavailable
     Resource.by_name('error.service_unavailable') || null_resource('error.service_unavailable')
   end
 
+  # @return [Resource, OpenStruct]
   def feedback
     Resource.by_name('ctas.feedback') || null_resource('ctas.feedback')
   end
 
+  # @return [Resource, OpenStruct]
   def content_footer
     Resource.by_name('ctas.content_footer') || null_resource('ctas.content_footer')
   end
 
+  # @return [Resource, OpenStruct]
   def other_useful_resources
     Resource.by_name('other_useful_resources') || null_resource('other_useful_resources')
   end
 
+  # @return [OpenStruct]
   def null_resource(name)
```

Comment:
We discussed naming consistency of `placeholder_<Object>`, i.e. `placeholder_resource` and `placeholder_asset` or similar.